src/Components/Faq.jsx

```diff
@@ -1,92 +0,0 @@
-"use client";
-import React, { useState, useRef } from "react";
-import { IoIosArrowDown } from "react-icons/io";
-
-const Faq = ({ title, title2, content, index, isOpen, onClick }) => {
```

Comment:
maybe use title and subtitle as your variables

I see no value the index is creating with this component